torch/onnx/symbolic_opset11.py

```diff
@@ -1062,12 +1062,15 @@ def linalg_vector_norm(g, self, ord, dim, keepdim, dtype):
             self = symbolic_helper._reshape_helper(
                 g, self, g.op("Constant", value_t=torch.tensor([-1], dtype=torch.int64))
             )
-            keepdim = None
+            keepdim = 0
+
         cond_op = g.op(
             "Not", g.op("Equal", self, g.op("Constant", value_t=torch.LongTensor([0])))
         )
         cond_op = g.op(
-            "Cast", cond_op, to_i=symbolic_helper.cast_pytorch_to_onnx["Long"]
+            "Cast",
+            cond_op,
+            to_i=symbolic_helper.cast_pytorch_to_onnx[self.type().scalarType()],
```

Comment:
This needs to be long because it is a accumulator

Reply:
Also prefer torch.int64 for clarity here and elsewhere

Reply:
> This needs to be long because it is a accumulator

1. From the PyTorch impl. it isn't limited to long and depends on the input scalar type.
2. Why should it be long as an accumulator? I did't hear this account before.
3. `ReduceSum` op allows a variety of dtypes (integers and floating points).
4. Maybe there is a better name suitable than `cond_op`.

Reply:
Sorry I confused this w/ another pr. SGTM